### PR TITLE
fix: forbidden update user

### LINKED ISSUE #2358: platform regular can't change personal settings
Any user should be able to change personal settings.
Screen Shot 2020-07-09 at 11 00 15 AM

### PR DESCRIPTION
**Pull request by wansir**
Signed-off-by: hongming <talonwan@yunify.com>

**What type of PR is this?**

 /kind bug

**What this PR does / why we need it**:

fix: update user info by self returns 403.

**Which issue(s) this PR fixes**:

Fixes #2358

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/kapis/iam/v1alpha2/handler.go [hunk 1]
```diff
@@ -1186,7 +1197,8 @@ func (h *iamHandler) updateGlobalRoleBinding(operator user.Info, user *iamv1alph
 		return err
 	}
 	if decision != authorizer.DecisionAllow {
-		err = errors.NewForbidden(iamv1alpha2.Resource(iamv1alpha2.ResourcesSingularUser), user.Name, fmt.Errorf("update global role binding not allowed"))
+		err = errors.NewForbidden(iamv1alpha2.Resource(iamv1alpha2.ResourcesSingularUser),
+			user.Name, fmt.Errorf("update global role binding not allowed"))
```
Comment (zryfish) [inline pkg/kapis/iam/v1alpha2/handler.go:1201]:
```suggestion
			user.Name, fmt.Errorf("update global role binding is not allowed"))
```

Comment (wansir) [reply pkg/kapis/iam/v1alpha2/handler.go]: fixed

**Pull request merged**